sdk/elixir/runtime/go.mod

```diff
@@ -2,40 +2,4 @@ module elixir-sdk
 
 go 1.22
 
-require (
```

Comment:
the go.mod changes were triggered by running `go mod tidy` in the runtime folder. I am trying to identify if there is a way to update the file without triggering these unwanted changes.